### PR TITLE
Feat/garbo mvp w routing

### PR DESCRIPTION
**Pull request by elvbom**

### CODE REVIEW

**Comment by Greenheart** [comment]
Shouldn't this PR be targeting the branch `feat/garbo-mvp-1` rather than `staging`? Then we can use `feat/garbo-mvp-1` as the base branch.

File: pages/_document.tsx [hunk 1]
```diff
@@ -1,79 +1,51 @@
...
-      })
-
-      const initialProps = await Document.getInitialProps(ctx)
-      return {
-        ...initialProps,
-        styles: (
-          <>
-            {initialProps.styles}
-            {sheet.getStyleElement()}
-          </>
```
Comment (Greenheart) [inline pages/_document.tsx]: The merge went wrong here. We need to generate the CSS code here.

File: next.config.js [hunk 2]
```diff
@@ -18,27 +28,14 @@ module.exports = withBundleAnalyzer({
...
-        source: '/kommuner/kommun/ume%C3%A5/parisavtalet',
-        destination: '/',
-        permanent: true,
-      },
-    ]
+    return paths.map((path) => ({
+      source: `/${path}/:slug*`,
+      destination: '/',
+      permanent: true,
+    }))
```
Comment (Greenheart) [inline next.config.js:39]: Looking good. I need to test it properly though.

**Review by Greenheart — CHANGES_REQUESTED** [review]

File: pages/[dataset]/index.tsx [hunk 3]
```diff
@@ -1,35 +0,0 @@
...
-    return {
-      notFound: true, // Return a 404 page when the dataset is not valid
-    }
-  }
-
-  return {
-    redirect: {
-      destination: `/${normalizedDataset}/${defaultDataView}`,
-      permanent: true,
-      shallow: true,
```
[the remark this answers is not in the document]
Comment (elvbom) [reply pages/[dataset]/index.tsx:28]: Now hardcoded in next-config

File: pages/index.tsx [hunk 4]
```diff
@@ -48,7 +48,18 @@ function StartPage({ companies, municipalities }: PropsType) {
   const [selectedDataset, setSelectedDataset] = useState<DatasetKey>(defaultDataset)
   const [selectedDataView, setSelectedDataView] = useState(normalizedDataView)
 
-  const [showRegionalEmissionData, setShowRegionalEmissionData] = useState(true)
+  const [showCompanyData, setShowCompanyData] = useState(true)
```
[the remark this answers is not in the document]
Comment (elvbom) [reply pages/index.tsx:51]: Sounds like a good idea! Would you like to fix in seperate PR when this is merged?

File: pages/_document.tsx (hunk 1 above)
Comment (elvbom) [reply pages/_document.tsx]: Meaning restore row 56-78?

File: next.config.js [hunk 5]
```diff
@@ -28,17 +39,10 @@ module.exports = withBundleAnalyzer({
...
```
Comment (Greenheart) [inline next.config.js:46]: This functions as intended by the code, always redirecting to `/` if you try to open an old route.

We discussed sending people to the start page so they could see the new company data. They can always navigate to the other datasets if that's what they are looking for.

This is ready to merge.

**Review by Greenheart — APPROVED** [review]

**Pull request merged**